### PR TITLE
[13.0][FIX] 'Release ready' based on the shipping policy of the order

### PR DESCRIPTION
**Pull request by sebalix**
Ref. 1671

### CODE REVIEW

**Review by TDu — APPROVED** [review]

File: sale_stock_available_to_promise_release/models/stock_picking.py [hunk 1]
```diff
@@ -10,3 +10,8 @@ class StockPicking(models.Model):
     sale_date_expected = fields.Date(
         string="Delivery Date", related="group_id.date_expected", store=True
     )
+
+    def _get_shipping_policy(self):
+        # Override the method to get the shipping policy from the related order
+        self.ensure_one()
+        return self.sale_id.picking_policy or super()._get_shipping_policy()
```
Comment (guewen) [inline sale_stock_available_to_promise_release/models/stock_picking.py:20]: Do you have an explanation why we have to look at the policy of the sale and not of the transfer?

Comment (simahawk) [reply sale_stock_available_to_promise_release/models/stock_picking.py:20]: @sebalix ping

Comment (sebalix) [reply sale_stock_available_to_promise_release/models/stock_picking.py:20]: updated

**Comment by sebalix** [comment]
@simahawk @guewen can you update your review please?

**Review by simahawk — APPROVED** [review]

**Review by guewen — APPROVED** [review]

**Pull request merged**